airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_manifest_only/pipeline.py

```diff
@@ -159,12 +159,14 @@ async def _run(self) -> StepResult:
         ## 2. Update the version in manifest.yaml
         try:
             manifest = read_yaml(root_manifest_path)
-            manifest["version"] = "4.3.0"
+            manifest["version"] = "4.3.2"
+            manifest["type"] = "DeclarativeSource"
```

Comment:
Use version 4.3.2 since that's the current version used in builder. Adding the type is required to prevent an early return from the propagate_types_and_parameters function